Export-GithubIssues-To-Spreadsheet/GithubToSpreadsheet/src/main/synapse-config/sequences/githubToSpreadsheet.xml

```diff
@@ -0,0 +1,143 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<sequence name="githubToSpreadsheet" trace="disable" xmlns="http://ws.apache.org/ns/synapse">
+    <dblookup>
+        <connection>
+            <pool>
+                <password>root</password>
+                <property name="autocommit" value="false"/>
+                <driver>com.mysql.jdbc.Driver</driver>
+                <url>jdbc:mysql://mysql.storage.cloud.wso2.com:3306/githubToSpreadsheetRegistry_testcompanybiru?useSSL=false</url>
+                <user>root_ZvVXAGb1</user>
```

Comment:
Is this always static?

Reply:
Is this always static?

Reply:
We can't pass values dynamically to dblookup mediator. Before they use the car file they need to modify with their credentials. I have documented this in the documentation.